### ISSUE TITLE
Error with: result = response.json()["choices"][0]["message"]["content"]

### ISSUE THREAD

**Issue by karpiyon** [larsgeb/vision-keywords#1]
Thanks for sharing:
Is it still working?

On this line, when I run in debug mode it see these error:

b'{\n    "error": {\n        "message": "The model `gpt-4-vision-preview` has been deprecated, learn more here: https://platform.openai.com/docs/deprecations",\n        "type": "invalid_request_error",\n        "param": null,\n        "code": "model_not_found"\n    }\n}\n'

**Comment by justinjonathan15** [comment]
i dont think it still works how written, I also get this error 404 Client Error: Not Found for url: https://api.openai.com/v1/completions
